docs/docs/forms.md

```diff
@@ -376,38 +379,41 @@ ReactDOM.render(<Form />, document.getElementById('root'));
 [Try this on CodePen.](https://codepen.io/ericnakagawa/pen/WGaYVg?editors=0010)
 
 
-### Basic Checkbox
+### Basic Uncontrolled Checkbox
 
 ```javascript
 class Form extends React.Component {
   constructor(props) {
     super(props);
-    this.state = {checked: ["B"]};
+    this.state = {checked: {"A": false, "B": true, "C": false}};
     this.handleChange = this.handleChange.bind(this);
     this.handleSubmit = this.handleSubmit.bind(this);
   }
 
   handleChange(event) {
-    let val = event.target.value;
-    let checked = this.state.checked.slice(); // copy
-    if(checked.includes(val)) {
-      checked.splice(checked.indexOf(val), 1);
-    } else {
-      checked.push(val);
-    }
+    let value = event.target.value;
+    let checked = this.state.checked; // copy 
+    if(!checked[value]) { checked[value] = true; } else { checked[value] = false; }
```

Comment:
can you put a space after `if` and break this up into multiple lines so it matches the style of other parts?